Demo/src/main/java/com/braintreepayments/demo/PayPalMessagingFragment.kt

```diff
@@ -0,0 +1,68 @@
+package com.braintreepayments.demo
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.LinearLayout
+import com.braintreepayments.api.PayPalMessagingListener
+import com.braintreepayments.api.PayPalMessagingLogoType
+import com.braintreepayments.api.PayPalMessagingOfferType
+import com.braintreepayments.api.PayPalMessagingRequest
+import com.braintreepayments.api.PayPalMessagingTextAlignment
+import com.braintreepayments.api.PayPalMessagingView
+
+class PayPalMessagingFragment : BaseFragment(), PayPalMessagingListener {
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_paypal_messaging, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val payPalMessagingRequest = PayPalMessagingRequest(
+            amount = 2.0,
+            placement = null,
+            offerType = PayPalMessagingOfferType.PAY_LATER_LONG_TERM,
+            buyerCountry = "US",
+            logoType = PayPalMessagingLogoType.PRIMARY,
+            textAlignment = PayPalMessagingTextAlignment.CENTER,
+            color = null
+        )
+
+        var payPalMessagingView = PayPalMessagingView(braintreeClient, requireActivity())
```

Comment:
Can we make this `var` a `val` here? It doesn't seem like we're resetting its value.